app/models/salt_handler/cloud_bootstrap.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+require "velum/salt"
+
+# This class is responsible to handle the salt events with tag "minion_start".
+# When such an event occurs, we want the minion to be saved in our database
+# if not already there.
+class SaltHandler::CloudBootstrap
+  attr_reader :salt_event, :job
+
+  def self.can_handle_event?(event)
+    matches = /^salt\/job\/(?<jid>\d+)\/ret\/admin$/.match(event.tag)
```

Comment:
`/^salt\/job\/(?<jid>\d+)\/ret\/admin$/` could be a constant on the class level, and also reused in the initializer

Reply:
Resolved in 7b1897cb1026b780a3a16bbd41625855365c49e4